src/CMakeLists.txt

```diff
@@ -332,51 +329,60 @@ if (MSVC)
 	)
 endif()
 
-add_executable(${EXECUTABLE} ${SOURCE})
-
-
-# Output the executable into the $/Server folder, so that it has access to external resources:
-set(EXECUTABLE_OUTPUT_PATH ${CMAKE_SOURCE_DIR}/Server)
-SET_TARGET_PROPERTIES(${EXECUTABLE} PROPERTIES
-	RUNTIME_OUTPUT_DIRECTORY_DEBUG          ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_RELEASE        ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_DEBUGPROFILE   ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_RELEASEPROFILE ${CMAKE_SOURCE_DIR}/Server
-)
+if (NOT ANDROID)
+	add_executable(${CMAKE_PROJECT_NAME} ${SOURCE})
```

Comment:
Naturally I have been in contact with tiger

He said he wanted to make sure it compiles. The current situation in cuberite/master is an Android App whicu provides an interface to a native library which provides an interface to the Android source. In my opinion this is just too complicated and it's much easier to just generate executables, but tiger stated on hangouts that this may take time and another pr. 'it doesn't fit the "make sure it builds" approach' 

As I am understanding this diff it's just disabling the executable for android which is at the moment the best we can to keep any usable wax for android and a base to build our new code on. I think it will take some time until android is finished in a way that we can use it like any other platform (running the actual binaries and such us a different problem, then, but one that is not hard to solve)